ElE-1668: Docs (#1169)
elementary-data/elementary: docs/guides/anomaly-detection-configuration/fail_on_zero.mdx

---

> title: "fail_on_zero"

Reviewer: This setting should only be applicable to volume anomalies, and needs to also appear in the volume anomalies parameter list [here](https://docs.elementary-data.com/guides/anomaly-detection-tests/volume-anomalies#test-configuration)

sidebarTitle: "fail_on_zero"
---

`fail_on_zero: true/false`

> Elementary anomaly detection tests will fail whether there is any zero metric value within the detection period.

Reviewer: ```suggestion
Elementary anomaly detection tests will fail if there is a zero metric value within the detection period.
```

If undefined, default is false.

> 

Reviewer: maybe we can add a graph? Do we have something on Figma?

Author: [https://github.com/elementary-data/elementary/pull/1169#discussion_r1336915277](Same)
